jwt-client/jwt-client-core/src/main/java/org/entur/jwt/client/properties/GenericJwtClientProperties.java

```diff
@@ -0,0 +1,31 @@
+package org.entur.jwt.client.properties;
+
+public class GenericJwtClientProperties extends AbstractJwtClientProperties {
+    protected String issuePath;
+    protected String refreshPath;
+    protected String revokePath;
+
```

Comment:
Perhaps we can add a  parameter for indicating whether or not to use the [authorization header or the request parameters ](https://www.oauth.com/oauth2-servers/access-tokens/client-credentials/) when requesting the JWT, seems both allowed but not required. 

```
protected String clientSecretSerializationFormat = "url-parameters"; // or "authorization-header"
```

or something like that, i.e. `credentialsFormat`  or `serializeCredentialsTo` or `clientSecretFormat` or `clientSecretRequestFormat`

Reply:
This is now implemented in the latest revision with the property name `clientSecretRequestFormat` accepting either `URL_PARAMETERS` or `AUTHORIZATION_HEADER`. In the application properties it can be set as:
```
entur.jwt.clients.generic.firstClient.client-secret-request-format=url-parameters
...
entur.jwt.clients.generic.secondClient.client-secret-request-format=authorization-header
```

Reply:
Looks great, but I see now that I made a faulty suggestion above, it should of course be `clientIdSecretSerializationFormat` with `Id` as the ingredients are `clientId` and `secret`. `clientIdSecretSerializationFormat` is somewhat hard on the eyes, so perhaps `credentialsRequestFormat` now looks better. @viliket what do you think?

Reply:
Yeah the current naming could still be misleading. Would `clientCredentialsRequestFormat` work for you?

Reply:
Yes

Reply:
Latest revision is now with this naming 👍